wpeverest/user-registration
# UR-275 Fix - Default reset password and myaccount design issue

### All Submissions:






### Changes proposed in this Pull Request:



Previously, The prevent core login feature doesn't restrict the default reset password i.e wp-login.php?action=resetpass and design issue with the vertical myaccount layout. This PR will these issues.

### How to test the changes in this Pull Request:

1. Go to the frontend and concate homepage url with /wp-login.php?action=resetpass or action=lostpassword and verify whether prevent core login is working or not.
2. Go to the General option User registration settings and then my account section. 
3. then select layout as vertical and save the setting.
4. Go to the frontend and then login and verify whether my account dashboard design is as expected or not.

### Types of changes:



### Other information:




### Changelog entry

> Fix - Default reset password and myaccount design issue.
